tasks/workflow/worker.go

```diff
@@ -314,19 +325,33 @@ func (w *worker) runCustomCommand(ctx context.Context, t *taskExecution) (contex
 	ctx = operations.SetOperationLogFields(ctx, op)
 	ctx = events.AddLogOptionalFields(ctx, events.LogOptionalFields{events.NodeID: nodeName, events.OperationName: op.Name})
 
+	executorOperationLabelDeployment := metrics.Label{
+		Name:  "Deployment",
+		Value: t.targetID,
+	}
+	executorOperationLabelName := metrics.Label{
+		Name:  "Name",
+		Value: op.Name,
+	}
+	executorOperationLabelNode := metrics.Label{
+		Name:  "Node",
+		Value: nodeType,
+	}
+	executorOperationLabels := []metrics.Label{executorOperationLabelDeployment, executorOperationLabelName, executorOperationLabelNode}
+
 	err = func() error {
-		defer metrics.MeasureSince(metricsutil.CleanupMetricKey([]string{"executor", "operation", t.targetID, nodeType, op.Name}), time.Now())
+		defer metrics.MeasureSinceWithLabels(metricsutil.CleanupMetricKey([]string{"executor", "operation"}), time.Now(), executorOperationLabels)
```

Comment:
like for delegate, I would call the metric `executor.operation` something like `executor.operation.duration` to describe it provides the duration of an execution